Doc/library/functions.rst

```diff
@@ -61,6 +61,22 @@ are always available.  They are listed here in alphabetical order.
    If the argument is a complex number, its magnitude is returned.
 
 
+.. function:: aiter(object, [sentinel])
+
+   Return an :term:`asynchronous iterator` object. This is the async variant
+   of the :func:`iter()` builtin, and behaves similarly.
```

Comment:
Fix these Sphinx things

Reply:
Are the links broken?

Reply:
With roles like `:func:` and `:meth:`, Sphinx adds the parens at the end for you when the docs are rendered. Fixed now.  Built the html docs locally with these changes and they look good to me.

Reply:
I think you can still include parentheses in the role text, though, right? I'm pretty sure I do that all the time, because it makes the reST more readable.

Reply:
I'm not sure if you can optionally include them. All the other occurrences of `:func:` in the documentation source do not include parens. At the very least, excluding them seems the preferred style.